packages/kbn-scout/src/playwright/runner/run_tests.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { resolve } from 'path';
+
+import { ToolingLog } from '@kbn/tooling-log';
+import { withProcRunner } from '@kbn/dev-proc-runner';
+import { getTimeReporter } from '@kbn/ci-stats-reporter';
+import { REPO_ROOT } from '@kbn/repo-info';
+import { runElasticsearch, runKibanaServer } from '../../servers';
+import { loadServersConfig } from '../../config';
+import { silence } from '../../common';
+import { RunTestsOptions } from './flags';
+import { getExtraKbnOpts } from '../../servers/run_kibana_server';
+
+export async function runTests(log: ToolingLog, options: RunTestsOptions) {
+  const runStartTime = Date.now();
+  const reportTime = getTimeReporter(log, 'scripts/playwright_test');
+
+  const config = await loadServersConfig(options.mode, log);
+  const playwrightConfigPath = options.configPath;
+
+  await withProcRunner(log, async (procs) => {
+    const abortCtrl = new AbortController();
+
+    const onEarlyExit = (msg: string) => {
+      log.error(msg);
+      abortCtrl.abort();
+    };
+
+    let shutdownEs;
+
+    try {
+      shutdownEs = await runElasticsearch({
+        onEarlyExit,
+        config,
+        log,
+        esFrom: options.esFrom,
+        logsDir: options.logsDir,
+      });
+
+      await runKibanaServer({
+        procs,
+        onEarlyExit,
+        config,
+        installDir: options.installDir,
+        extraKbnOpts: getExtraKbnOpts(options.installDir, config.get('serverless')),
+      });
+
+      // wait for 5 seconds
+      await silence(log, 5000);
```

Comment:
are we doing this to make sure Kibana and ES are connected to each other? If thats the case, can we find a better way of doing it like making an http request or so?

Reply:
I think we might not even need it, but it was copied for `kbn/test`. I think Robert plans to look into Kibana loading state check and maybe we put something here. I will test it with the build, maybe we are good without.

Reply:
Update: we decided to keep it for now for consistency with FTR and will address it later in `kbn-test` package